azero/contracts/most/lib.rs

```diff
@@ -436,12 +435,12 @@ pub mod most {
             Ok(self.data()?.request_nonce)
         }
 
-        /// Query pocket money
+        /// Query subsidy contract
         ///
-        /// An amount of the native token that is tranferred with every request
+        /// An address of the contract that may be used to subsidize bridging users
         #[ink(message)]
-        pub fn get_pocket_money(&self) -> Result<Balance, MostError> {
```

Comment:
names are all over the place. Contract is called `money_box`. In the callee contract it's `subsidy_contract`. Constant is `MONEY_BOX_GAS_LIMIT`. Please make them uniform